src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import Page from './components/Page';
+
+function App() {
+  const [state, setState] = useState({
+    count: 0,
+  });
+
+  const { count } = state;
+
+  function handleClick() {
+    setState({
+      count: count + 1,
+    });
+  }
+
+  function handleClickNumber(value) {
+    setState({
+      count: count + value,
+    });
+  }
+
+  return (
+    <Page count={count} onClick={handleClick} onClickNumber={handleClickNumber} />
```

Comment:
```suggestion
    <Page 
      count={count}
      onClick={handleClick}
      onClickNumber={handleClickNumber}
    />
```

어떤게 더 나은가요?
저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.

Reply:
속성 2개!! 저도 이 기준에 따라 구분하면 좋을 것 같아요!! 감사합니다 ㅎㅎ